transport/http/server.go

```diff
@@ -112,6 +113,19 @@ func Listener(lis net.Listener) ServerOption {
 	}
 }
 
+// handleFuncWrapper is a wrapper for http.HandlerFunc to implement http.Handler
```

Comment:
Don't add this. should use http.handler and http.handlerfunc

Reply:
ok